Userland/Libraries/LibGUI/Widget.cpp

```diff
@@ -47,6 +47,7 @@ Widget::Widget()
     REGISTER_BOOL_PROPERTY("visible", is_visible, set_visible);
     REGISTER_BOOL_PROPERTY("focused", is_focused, set_focus);
     REGISTER_BOOL_PROPERTY("enabled", is_enabled, set_enabled);
+    REGISTER_BOOL_PROPERTY("accepts_emoji", accepts_emoji_input, set_accepts_emoji_input);
```

Comment:
Look like this property isn't actually used in GML in this PR, so just a heads up, this is going away in a PR to support multi-code point emoji

Reply:
rgr I'll drop it